backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -125,6 +125,39 @@ public UpdateShopStatusResponse updateShop(UpdateShopStatusRequest updateShopSta
 		return new UpdateShopStatusResponse(shopMap.get(updateShopStatusRequest.getShopId()));
 	}
 
+	public Token getTokenByTokenId(UUID tokenId) {
```

Comment:
Nit. We've pointed similar things earlier in some other functions.

Shall we name this "getToken" ? since tokenId is already being passed in the param, it's redundant in the method name.

Reply:
Yes Yes. Changed it.